style.css

```diff
@@ -20,385 +20,974 @@ nav {
   grid-template-columns: 100%;
   grid-auto-rows: minmax(552px, auto);
   grid-gap: 10px;
-  width: 100%;
 }
 
-.worksec1 {
-  position: static;
-  width: 88%;
-  height: 310px;
-  margin: 25px;
-  left: 50%;
-  top: 40%;
-  border-radius: 9px;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  background: white;
-  padding-bottom: 142px;
-  padding-top: 76px;
-  display: flex;
-  flex-direction: row;
-}
+@media only screen and (max-width: 992px) {
+  /* for phone  * */
```

Comment:
- [ ] make sure that you use [the mobile-first design](https://medium.com/@Vincentxia77/what-is-mobile-first-design-why-its-important-how-to-make-it-7d3cf2e29d00) approach while implementing the desktop version. For your project, we can see that for a max-width more than 780px, which is supposed to be for desktop, your design still looks like for mobile version.  
**Your template** :
![image](https://user-images.githubusercontent.com/71874425/154572915-301417ae-e442-45aa-a19f-299d44faaffd.png)

 
I'm really sorry, I couldn't approve your project while it looks again like this. to fix it, use in your media queries (`min-width: 992px`) instead of (`max-width:992px`). The value for min-width property should be in range of `992px` - `1100px`